test/versioned/prisma/prisma/schema.prisma

```diff
@@ -1,6 +1,7 @@
 generator client {
   provider        = "prisma-client-js"
   previewFeatures = ["tracing"]
+  binaryTargets   = ["native"]
```

Comment:
has this changed over the 4.x line because looking at their docs this is just being redundant

> Note: The native binary target is the default. You can set it explicitly if you wish to

Reply:
idk, but the build is green again. Think it'll stay green if we merge?